### DISCUSSION TITLE
kstars-3.6.0.dmg SHA256 mismatch

### DISCUSSION THREAD

**Discussion by chrischram** [Homebrew/discussions#3636] [Casks]
brew install --cask kstars
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).

==> Downloading https://www.indilib.org/jdownloads/kstars/kstars-3.6.0.dmg
Already downloaded: /Users/cschram/Library/Caches/Homebrew/downloads/31515ca1c120a1058e199ed3f5256323afc2e81c8c193323811aabec0d9d3110--kstars-3.6.0.dmg
Error: SHA256 mismatch
Expected: 2c4c72a39b6320c68341b42f0cce3a8cf28fde188e5184da40c5af127bd2be23
  Actual: 57b385e1d4dba46315b74e85566689bd1a69331be10c816b671c9d63c822c1b7
    File: /Users/cschram/Library/Caches/Homebrew/downloads/31515ca1c120a1058e199ed3f5256323afc2e81c8c193323811aabec0d9d3110--kstars-3.6.0.dmg
To retry an incomplete download, remove the file above.

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/131373

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match
